core/store/tx_manager_test.go

```diff
@@ -485,16 +489,15 @@ func TestTxManager_BumpGasUntilSafe_confirmed(t *testing.T) {
 	gasThreshold := sentAt + config.EthGasBumpThreshold()
 	minConfs := config.MinOutgoingConfirmations() - 1
 	require.NoError(t, app.Store.ORM.CreateHead(cltest.Head(gasThreshold+minConfs-1)))
-	ethMock, err := app.MockStartAndConnect()
-	require.NoError(t, err)
+	require.NoError(t, app.StartAndConnect())
```

Comment:
Maybe put StartAndConnect onto its own line?  The pattern should probably be moved away from the code base, it would be clearer.

Reply:
Not sure I follow @felder-cl - do you mean add a newline above?

Reply:
I find this clearer ->
```
err = app.MockStartAndConnect()
require.NoError(t, err)
```